url/urltestdata.txt

```diff
@@ -258,6 +258,9 @@ test.txt  s:http h:www.example.com p:/test.txt
 \u4E2D/test.txt  s:http h:www.example.com p:/%E4%B8%AD/test.txt
 http://www.example2.com  s:http h:www.example2.com p:/
 //www.example2.com  s:http h:www.example2.com p:/
+file:...  s:file h: p:/...
+file:..  s:file h: p:/..
```

Comment:
# Why The Web Is Great™

a short novel by @frewsxcv

## Chrome

![screen shot 2015-08-12 at 6 41 31 pm](https://cloud.githubusercontent.com/assets/416575/9238619/eecfc708-4121-11e5-8e44-d153bad69551.png)

## Firefox

![screen shot 2015-08-12 at 6 43 00 pm](https://cloud.githubusercontent.com/assets/416575/9238632/006f9dc6-4122-11e5-94ed-8470b16cb78f.png)